### ISSUE TITLE
[App] Frozen line plot for over 10,000 steps

### ISSUE THREAD

**Issue by andreimargeloiu** [wandb/client#3150]
**Describe the bug**
In some cases, after a long time, the line plot of the validation accuracy in a remains constant at 0.5 for 10,000 steps. This is suspicious, because the train accuracy, as well as **train and valid losses, continue to update.**

![CleanShot 2022-01-17 at 11 04 57](https://user-images.githubusercontent.com/18227298/149758555-a0e5f2b2-6840-4c82-846a-6b3f5e7cbe4b.png)

![CleanShot 2022-01-17 at 11 06 03](https://user-images.githubusercontent.com/18227298/149758722-39adbd39-2a1a-4738-8765-63a20afcbecd.png)

Because the validation loss continues to change significantly, I wonder why the validation is changing even with 0.0001 over so many steps. Are there current relevant wandb known issues?

**Operating System**
Logs made directly from pytorch-lightning==1.5.8 with wandb=0.12.9, pytorch 1.10.1, python 3.7.9

**Comment by armanhar** [comment]
Hey Andrei, is this something that has happened multiple times with your charts? Can you share your browser's console output?

**Comment by andreimargeloiu** [comment]
It happened only once. Sorry, I ran a few hundred more experiments and can't find this one.

**Comment by armanhar** [comment]
Doesn't seem like there are any wandb errors, Andrei. I'll close the ticket for now but please don't hesitate to ping us if this happens again.
